src/TimeoutMigrationTool/RavenDB/RavenDBTimeoutStorage.cs

```diff
@@ -129,7 +130,7 @@ internal async Task<List<BatchInfo>> PrepareBatchesAndTimeouts(DateTime maxCutof
 
             bool filter(TimeoutData td)
             {
-                return td.Destination == endpoint.EndpointName && td.Time >= maxCutoffTime && !td.OwningTimeoutManager.StartsWith(RavenConstants.MigrationOngoingPrefix,
+                return td.OwningTimeoutManager == endpoint.EndpointName && td.Time >= maxCutoffTime && !td.OwningTimeoutManager.StartsWith(RavenConstants.MigrationOngoingPrefix,
```

Comment:
In case the OwningTimeoutManager has the prefix set, then the OwningTimeoutManager will never match the endpointname, so we can remove the last condition. Do we need to consider any usecases in which we want to take into account the timeouts that have the owningtimeoutmanager set to prefix_orginalvalue?

Reply:
I have a hunch that this change is going to invalidate some of the assumptions we made so far.